client/state/countries/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getCountries( state, type ) {
+	return state.countries.items[ type ];
+}
+
+export function isFetching( state, type ) {
```

Comment:
Ok that makes sense, I'm changing things towards @gwwar 's suggestions, which will make things more self explanatory, then we can check if something needs a doc and I'll add :)